.github/workflows/zip.yml

```diff
@@ -422,7 +416,7 @@ jobs:
     - name: Move frameworks
       run: |
         mkdir -p "${HOME}"/ios_frameworks/
-        find "${GITHUB_WORKSPACE}/${FRAMEWORK_DIR}" -name "Firebase*latest.zip" -exec unzip -d "${HOME}"/ios_frameworks/ {} +
+        find "${GITHUB_WORKSPACE}" -name "Firebase*latest.zip" -exec unzip -d "${HOME}"/ios_frameworks/ {} +
```

Comment:
@ncooke3 Would you confirm this structure change is ok for the release process?

Reply:
The directory structure of the the downloadable release artifacts looks the same.